riptide-core/src/test/java/org/zalando/riptide/RequesterTest.java

```diff
@@ -114,6 +114,28 @@ public void shouldEncodeAppendedQueryParams() {
                         on(SUCCESSFUL).call(pass()));
     }
 
+    @Test
+    public void shouldAppendedDateTimeQueryParams() {
+        expectRequestTo("https://test.datetimes.org/index.php?to=2018-05-21T10%3A24%3A47.788&from=2017-04-20T09%3A23%3A46.787");
+
+        unit.head("https://test.datetimes.org/index.php")
+                .queryParam("to", "2018-05-21T10:24:47.788")
```

Comment:
Not relevant for the test, but proper ISO 8601 has no milliseconds and it has to have `Z` for UTC or an offset.